cosmovisor/args_test.go

```diff
@@ -158,11 +163,27 @@ func (s *argsTestSuite) TestValidate() {
 		valid bool
 	}{
 		"happy": {
-			cfg:   Config{Home: absPath, Name: "bind"},
+			cfg:   Config{Home: absPath, Name: "bind", DataBackupPath: absPath},
 			valid: true,
 		},
 		"happy with download": {
-			cfg:   Config{Home: absPath, Name: "bind", AllowDownloadBinaries: true},
+			cfg:   Config{Home: absPath, Name: "bind", AllowDownloadBinaries: true, DataBackupPath: absPath},
+			valid: true,
+		},
+		"happy with skip data backup": {
+			cfg:   Config{Home: absPath, Name: "bind", UnsafeSkipBackup: true, DataBackupPath: absPath},
+			valid: true,
+		},
+		"happy with skip data backup and empty data backup path": {
+			cfg:   Config{Home: absPath, Name: "bind", UnsafeSkipBackup: true, DataBackupPath: ""},
+			valid: true,
+		},
+		"happy with skip data backup and no such data backup path dir": {
+			cfg:   Config{Home: absPath, Name: "bind", UnsafeSkipBackup: true, DataBackupPath: filepath.FromSlash("/no/such/dir")},
+			valid: true,
+		},
+		"happy with skip data backup and relative data backup path": {
+			cfg:   Config{Home: absPath, Name: "bind", UnsafeSkipBackup: true, DataBackupPath: relPath},
```

Comment:
positive test cases are added here, @robert-zaremba, presently, if `UnsafeSkipBackup == true`, then we allow invalid `DataBackupPath` , is this design ok?

Reply:
Yes, this is reasonable.